packages/wonder-blocks-link/components/link.js

```diff
@@ -124,7 +124,12 @@ export default class Link extends React.Component<SharedProps> {
         );
 
         return (
-            <ClickableBehavior disabled={false} onClick={onClick} href={href}>
+            <ClickableBehavior
+                disabled={false}
+                onClick={onClick}
+                href={href}
+                triggerOnSpace={false}
```

Comment:
I was thinking we could replace `triggerOnSpace` and `triggerOnEnter` with `role`.  This prop would  take only those aria roles ("button", "link", etc.) that can be activated by pressing space or enter.  `ClickableBehavior` would be responsible for making the decision (based on that prop) about when to trigger the `onClick` handler.  We could even pass the `role` as part of `handlers` which should probably be renamed `childProps` since it also include `tabindex`.

This is part of a bigger plan I have to allow standard aria props (including `role`) on more of our components.  Obviously not all components need `role` since for `Button` the role should always be button, but being able to set `aria-label`, `aria-labelledby`, etc. on things in a way that matches how it's done on tags should be a win.

I've been meaning to write this all up in a doc, but haven't had time.

Reply:
I'm kind of worried that the list that I create wouldn't be exhaustive and since ClickableBehavior is exported as a component to use in webapp, if someone tries to use it for a non-existent but valid role, ClickableBehavior would have to be updated here and then republished.

I'm also worried about this PR getting too big, so it would be more optimal for me to merge this and work on role-related behavior separately.

If there's a concern about the `trigger` props leaking into webapp before the role stuff could be added, I can set `@ignore` on them again.

Reply:
Yeah let's split off the `role` work.  I terms of people using it for unsupported roles we can document which roles it supports and expand it as necessary.  TBH, I don't see people using this component directly very often in webapp.